Update license-and-subscription.rst (#4447)
mattermost/docs: source/overview/license-and-subscription.rst

The United States has a number of export control regulations it has implemented to protect national security interests and to promote its foreign policy objectives. Based on these regulations, U.S. companies are prohibited from doing business with specific countries which have been embargoed by the U.S. government. They are also prohibited from exporting certain items to certain countries that have been sanctioned by the U.S. government. In addition, U.S. companies are prohibited from doing business with specific people and/or companies that have been named and listed by the U.S. government. 

In order to comply with these requirements, Mattermost must collect the name and physical address of all individuals and companies it does business with so that it can comply with the U.S. export controls regulations.

What does Mattermost do with this information?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The information you provide is used for a screening process. There are two different purposes for screening: 

> - One screening is to ensure against exports of certain restricted goods to countries that are embargoed. In our case, it is the software that has encryption in it.

Reviewer: ```suggestion
- One screening is to ensure against exports of certain restricted goods to countries that are embargoed. In our case, goods refer to our software that has encryption in it.
```

> - The other screening is against people and companies. There are certain people and companies that the government has put on a list (the Denied Party List) that US companies are prohibited from doing any business with for various reasons. They could be terrorists, on a terrorist watch list, they could help finance terrorists, they could participate in human rights violations, etc. If they are on the Denied Party List, we are not able to do any business with them.

Reviewer: ```suggestion
- The other screening is against people and companies. There are certain people and companies that the government has put on a list (the Denied Party List) that US companies are prohibited from doing any business with for various reasons. They could be terrorists, be on a terrorist watch list, could be helping finance terrorists, could be participating in human rights violations, etc. If they are on the Denied Party List, we are not able to do any business with them.
```

Should you wish to make payment using another method, please contact our `Billing team <mailto:AR@mattermost.com>`_.

> Why do I need to provide my physical address?

Reviewer 2: ```suggestion
Why do I need to provide my physical address when purchasing a Self-Managed Enterprise license?
```

Reviewer 2: @jamiehurewitz per our earlier conversation, this applies for self-managed licenses correct? Because for Cloud, we're basing it on the billing/payment address.

https://community.mattermost.com/private-core/pl/duojwkwzc3ds5cys7gr4a5nskh

Reviewer 3: That is not necessarily true. We use the address also for back up research purposes as well. For example, if we get a hit on a Jason Blais, and the customer is located in Canada but the individual that is on the denied party list which caused the hit, is actually located in France, we use the address to help determine whether the hit we got is a legitimate hit and are unable to proceed with the transaction or if it is just someone with the same name, thus a false flag and can clear the transaction to proceed.

Reviewer 2: @jamiehurewitz Makes sense, but we are not collecting the physical address for Cloud transactions though. So I just want to make sure our documentation is accurate.

We could change the title to `Why do I need to provide my name and physical address?` in which case the answer would apply to cloud and self-managed.

Reviewer 3: I will leave it up to you to decide. I just wanted you to understand how the information is used.

Author: I've edited your suggestion, @jasonblais - I think it works well based on the thread above.

Who are the sanctioned people, companies, and entities?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The Office of Foreign Assets Control (OFAC) maintains a list of sanctioned entities. Some examples include:

- Terrorists
- Banks or other financial institutions that are involved in financing terrorism
- Companies or people that have been involved in human or drug trafficking
- Organizations that have been sanctioned for human rights violations

This will also include people in violation of government contracts because of our business with the U.S. government. Individuals and Companies do not end up on the Denied Party List based on the country they live in but by their actions and conduct.

Is my information shared with any third-parties?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> A third-party tool is used for the screening process, but your details are not stored by them.

Author: @jasonblais, can you confirm whether this is correct?

Reviewer 3: This information is not accurate. The screening information is captured and stored to ensure we can back up any audit of our export compliance.

Author: Thanks @jamiehurewitz - I thought that was the case. In order not to create confusion, can we link to the third-party's site to answer this question (specifically to their compliance page)? Or do we store the information?

Reviewer 3: Yes I can certainly find this information. The additional content on the bottom half was not drafted by me. This was added by @jasonblais. I will defer to how he would like it to be presented since this information is not required from a privacy standpoint. We are collecting the information for legal requirements.

Author: In that case, I think this question can be removed as it actually creates more questions. If the issue does arise, we can address it.

Reviewer 2: @justinegeffen yea, let's remove at this time

> What does “shipping address” mean for software that will be used in many places?

Reviewer 2: ```suggestion
What does “physical address” mean for software that will be used in many places?
```

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> In this case, the "shipping address" is the location where the individual, who will be receiving the license key, is physically located and will be able to access the software for installation.

Reviewer 2: ```suggestion
In this case, the "physical address" is the location where the individual, who will be receiving the license key, is physically located and will be able to access the software for installation.
```

> In this case, the "shipping address" is the location where the individual, who will be receiving the license key, is physically located and will be able to access the software for installation.

Author: ```suggestion
In this case, the "physical address" is the location where the individual, who will be receiving the license key, is physically located and will be able to access the software for installation.
```
